core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala

```diff
@@ -1653,7 +1663,19 @@ private[spark] object JsonProtocolSuite extends Assertions {
       |      "Remote Bytes Read": 1000,
       |      "Remote Bytes Read To Disk": 400,
       |      "Local Bytes Read": 1100,
-      |      "Total Records Read": 10
+      |      "Total Records Read": 10,
+      |      "Remote Requests Duration": 900,
+      |      "Push Based Shuffle": {
+      |         "Corrupt Merged Block Chunks" : 100,
+      |         "Merged Fetch Fallback Count" : 100,
+      |         "Merged Remote Blocks Fetched" : 0,
+      |         "Merged Local Blocks Fetched" : 0,
+      |         "Merged Remote Chunks Fetched" : 0,
+      |         "Merged Local Chunks Fetched" : 0,
+      |         "Merged Remote Bytes Read" : 0,
+      |         "Merged Local Bytes Read" : 0,
+      |         "Merged Remote Requests Duration": 900
```

Comment:
If merged remote blocks are 0 then this can't be > 0

Reply:
This is [currently being set](https://github.com/apache/spark/blob/e46031651617393bb182e44ae04920d6c89592f6/core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala#L1315) to `executorDeserializeTime` + `jvmGCTime` while making the test task metrics, do you think that assumption is incorrect ?

Reply:
merge remote request duration is not at all related to `executorDeserializeTime` and `jvmGCTime`. I guess we didn't pay attention to these tests internally.